samples/peripheral/802154_sniffer/prj.conf

```diff
@@ -5,7 +5,7 @@ CONFIG_IEEE802154_RAW_MODE=y
 CONFIG_NET_PKT_TIMESTAMP=y
 
 # Allow sharing the RTC between IEEE 802.15.4 and Zephyr
-CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=3
+CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=2
```

Comment:
Again valid for `CONFIG_SOC_NRF5340_CPUNET` only, not in general.

Reply:
True, reverted.

However the `nrf5340dk_nrf5340_cpunet` board is not supported for this sample, see https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/peripheral/802154_sniffer/README.html

So no changes should be introduced here.